CHANGELOG.md

```diff
@@ -12,6 +12,12 @@
     - `-querier.query-ingesters-within`
     - `-querier.query-store-after`
 * [CHANGE] Config flag category overrides can be set dynamically at runtime. #1934
+* [CHANGE] Distributor: Added limit to prevent tenants from sending excessive number of requests: #1843
```

Comment:
Since the default value is 0 now (disabled) then this is not a CHANGE but ENHANCEMENT.